README.md

```diff
@@ -1,54 +1,52 @@
 
-## Get started
+### How It Works
 
-If you already have an understanding of the tech stack, use our [Get Started](docs/get-started.md) guide.
+**RLN (Rate Limiting Nullifier)**: A cryptographic system that prevents spam by limiting transaction rates through nullifier-based proofs. Implementation can be found [here](https://github.com/vacp2p/status-rln-prover).
 
-For developers looking to build services locally (e.g., the coordinator), see our detailed [Local Development Guide](docs/local-development-guide.md).
+**火 Karma System**: A reputation-based mechanism where users earn Karma soulbound tokens through positive network participation. Users will have different levels of daily gasless transaction quota depending on their Karma amount. Contract code implementation can be found [here](https://github.com/vacp2p/staking-reward-streamer).
 
-## Looking for the Linea code?
+### Key Features
 
-Linea's stack is made up of multiple repositories, these include:
+- **🆓 Zero Gas Transactions**: Users with sufficient Karma balance can submit transactions without paying gas
+- **🛡️ Rate Limiting**: RLN-based rate limiting prevents spam and abuse through cryptographic proofs
+- **💰 Premium Gas Bypass**: Users who exceed their quota can bypass restrictions with premium gas payments
+- **📊 Reputation-Based Access**: Karma tokens determine user privileges and transaction quotas
 
-- [linea-monorepo](https://github.com/Consensys/linea-monorepo): The main repository for the Linea stack & network
-- [linea-besu](https://github.com/Consensys/linea-besu): Fork of Besu to implement the Linea-Besu client
-- [linea-sequencer](https://github.com/Consensys/linea-sequencer): A set of Linea-Besu plugins for the sequencer and RPC nodes
-- [linea-tracer](https://github.com/Consensys/linea-tracer): Linea-Besu plugin which produces the traces that the constraint system applies and that serve as inputs to the prover
-- [linea-constraints](https://github.com/Consensys/linea-constraints): Implementation of the constraint system from the specification
-- [linea-specification](https://github.com/Consensys/linea-specification): Specification of the constraint system defining Linea's zkEVM
+### Architecture Components
 
-Linea abstracts away the complexity of this technical architecture to allow developers to:
+#### Prover Components
+- [**TxForwarder Transaction Pool Validator**](besu-plugins/linea-sequencer/sequencer/src/main/java/net/consensys/linea/sequencer/txpoolvalidation/validators/RlnProverForwarderValidator.java): Forwards incoming transaction data to the RLN prover service to generate RLN proofs
+- [**Enhanced LineaEstimateGas RPC**](besu-plugins/linea-sequencer/sequencer/src/main/java/net/consensys/linea/rpc/methods/LineaEstimateGas.java): Dynamically provides zero gas or premium gas estimates based on real-time user karma and usage quotas
```

Comment:
> Enhanced LineaEstimateGas RPC

personally don't like the word enhanced here, modified sounds better

Reply:
Linea estimate gas rpc is more appropriately part of sequencer or if you decide to make a section for RPC mode of besu specifically, it'll fit there

Reply:
[a2b9fc0](https://github.com/status-im/status-network-monorepo/pull/4/commits/a2b9fc0460b6640c013c5b7cd4ebf60c903d55d2)
Enhanced -> Modified